### PR TITLE
Fixed dag and drop .length error

### PR DESCRIPTION
**Pull request by jobpaardekooper**
If you drag an image into the draft-js editor it throws an error:

```
insertTextIntoContentState.js:25 Uncaught TypeError: Cannot read property 'length' of null
    at insertTextIntoContentState (insertTextIntoContentState.js:25)
    at Object.replaceText (DraftModifier.js:63)
    at Object.insertText (DraftModifier.js:67)
    at insertTextAtSelection (DraftEditorDragHandler.js:116)
    at onDrop (DraftEditorDragHandler.js:102)
    at DraftEditor.react.js:268
    at HTMLUnknownElement.callCallback (react-dom.development.js:147)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:196)
    at invokeGuardedCallback (react-dom.development.js:250)
    at invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:265)
    at executeDispatch (react-dom.development.js:571)
    at executeDispatchesInOrder (react-dom.development.js:596)
    at executeDispatchesAndRelease (react-dom.development.js:695)
    at executeDispatchesAndReleaseTopLevel (react-dom.development.js:704)
    at forEachAccumulated (react-dom.development.js:676)
    at runEventsInBatch (react-dom.development.js:844)
    at runExtractedEventsInBatch (react-dom.development.js:852)
    at handleTopLevel (react-dom.development.js:5030)
    at batchedUpdates$1 (react-dom.development.js:21469)
    at batchedUpdates (react-dom.development.js:2247)
    at dispatchEvent (react-dom.development.js:5110)
    at react-dom.development.js:21526
    at Object.unstable_runWithPriority (scheduler.development.js:255)
    at interactiveUpdates$1 (react-dom.development.js:21525)
    at interactiveUpdates (react-dom.development.js:2268)
    at dispatchInteractiveEvent (react-dom.development.js:5086)
insertTextIntoContentState @ insertTextIntoContentState.js:25
replaceText @ DraftModifier.js:63
insertText @ DraftModifier.js:67
insertTextAtSelection @ DraftEditorDragHandler.js:116
onDrop @ DraftEditorDragHandler.js:102
(anonymous) @ DraftEditor.react.js:268
callCallback @ react-dom.development.js:147
invokeGuardedCallbackDev @ react-dom.development.js:196
invokeGuardedCallback @ react-dom.development.js:250
invokeGuardedCallbackAndCatchFirstError @ react-dom.development.js:265
executeDispatch @ react-dom.development.js:571
executeDispatchesInOrder @ react-dom.development.js:596
executeDispatchesAndRelease @ react-dom.development.js:695
executeDispatchesAndReleaseTopLevel @ react-dom.development.js:704
forEachAccumulated @ react-dom.development.js:676
runEventsInBatch @ react-dom.development.js:844
runExtractedEventsInBatch @ react-dom.development.js:852
handleTopLevel @ react-dom.development.js:5030
batchedUpdates$1 @ react-dom.development.js:21469
batchedUpdates @ react-dom.development.js:2247
dispatchEvent @ react-dom.development.js:5110
(anonymous) @ react-dom.development.js:21526
unstable_runWithPriority @ scheduler.development.js:255
interactiveUpdates$1 @ react-dom.development.js:21525
interactiveUpdates @ react-dom.development.js:2268
dispatchInteractiveEvent @ react-dom.development.js:5086
```
With this change the editor checks if there is any text and if there is it handles it like normal, if there isn't it does nothing.

Closes: #2116

### CODE REVIEW

**Comment by jobpaardekooper** [comment]
This is also a proper fix for: #291.

File: src/model/transaction/insertTextIntoContentState.js [hunk 1]
```diff
@@ -32,7 +32,11 @@ function insertTextIntoContentState(
     '`insertText` should only be called with a collapsed range.',
   );
 
-  const len = text.length;
+  let len;
```
Comment (claudiopro) [inline src/model/transaction/insertTextIntoContentState.js]: Can you please declare the Flow type and initialize the variable?

```js
let len: ?number = null;
```

File: src/model/transaction/insertTextIntoContentState.js [hunk 2]
```diff
@@ -32,7 +32,11 @@ function insertTextIntoContentState(
...
+  if (text) {
```
Comment (claudiopro) [inline src/model/transaction/insertTextIntoContentState.js]: Please make the check more explicit
```js
if (text != null) {
```

File: src/model/transaction/insertTextIntoContentState.js [hunk 3]
```diff
@@ -32,7 +32,11 @@ function insertTextIntoContentState(
...
+    len = text.length;
+  }
+
   if (!len) {
```
Comment (claudiopro) [inline src/model/transaction/insertTextIntoContentState.js]: I would also make this check more explicit as `len` can be `null` or `0`.
```js
if (len == null || len === 0) {
```
Note the non-strict equality check `len == null` that is also true if `len` is `undefined`.

**Comment by jobpaardekooper** [comment]
Also correctly addresses #1293 and #494.

**Comment by jobpaardekooper** [comment]
@claudiopro thank you for the tips and the reasoning behind them!

**Pull request closed without merging**